exporter/prometheusexporter/sanitize.go

```diff
@@ -35,7 +35,12 @@ func sanitize(s string) string {
 	if unicode.IsDigit(rune(s[0])) {
 		s = "key_" + s
 	}
-	if s[0] == '_' {
+	//replace labels starting with _ only when sanitizeLabel is disabled
```

Comment:
is there a reason for using the disabling of `sanitizeLabel` here? if not, it would be easier to read and understand the following code as:

```
if sanitizeLabel && s[0] == '_' {
```

Reply:
by default sanitizeLabel will be disabled(false).

if sanitizeLabel is disabled and the label starts with `_`, we must add `key_` to the label.

Reply:
Right... this is the confusing part.

Saying that `sanitizeLabel` being disabled will modify (or sanitize) the label id what's confusing here.

Reply:
updated this part. please review.